app/mailers/mailer.rb

```diff
@@ -1,11 +1,8 @@
-class CommunicartMailer < ApplicationMailer
+class Mailer < ApplicationMailer
```

Comment:
Hi @JuanitoFatas -- we do have an `ApplicationMailer` in this project.

Our other mailers inherit from it and it basically just contains helper methods at the moment. 

I'd rather rename this mailer to `Mailer` and then move stuff out of it into more specific mailers than put everything in ApplicationMailer since it's nice to be able to easily navigate that file in order to find helper the methods.

Thank you for weighing in!